source/administration/config-settings.rst

```diff
@@ -2681,27 +2681,30 @@ So you don't miss messages, please make sure to change this value to an email yo
 | This feature's ``config.json`` setting is ``"SupportEmail":"feedback@mattermost.com"`` with string input.                                                            |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Enable Custom Terms of Service (Beta)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Custom Terms of Service
+~~~~~~~~~~~~~~~~~~~~~~~~~
 *Available in Enterprise Edition E20*.
 
-**True**: New users must accept custom terms of service before accessing any Mattermost teams on desktop or web. Existing users must accept them after login or a page refresh. Users on mobile will not be presented with the custom Terms of Services. Mobile support is scheduled for an upcoming release.
-
-**False**: During account creation or login, users can review terms of service included via **System Console > Legal and Support > Terms of Service link**.
+Enable Custom Terms of Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note::
 
-  This setting can only be modified using the System Console user interface.
+  This page can only be modified using the System Console user interface.
 
-Custom Terms of Service Text (Beta)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-*Available in Enterprise Edition E20*.
+**True**: When true, new users must accept the terms of service before accessing any Mattermost teams on desktop, web or mobile. Existing users must accept them after login or a page refresh. To update terms of service link displayed in account creation and login pages, go to **System Console > Legal and Support > Terms of Service Link**.
+
+**False**: During account creation or login, users can review terms of service included via **System Console > Legal and Support > Terms of Service link**.
 
```

Comment:
This was a bit confusing to me on what the difference was between the two settings.  Would we recommend them to be similar or the same material when the setting is enabled?  Maybe we could clarify the behavior a bit more of the account creation/login Terms of Service Link - "During account creation or login, users can review terms of service by accessing the link configured via..." 0/5.  I had to read this section a couple of times to figure out what it was saying.

Reply:
Yeah, it's a bit weird right now. Ideally the terms of service that an admin updates would automatically update the text that would be displayed by Terms of Service link. But that would have been considerable effort.

Right now the expectation is that if you update your custom terms of service, you would also update it in the internal site that you're linking to in Terms of Service.